### DISCUSSION TITLE
Support multipart/from-data on Flow POST Trigger

### DISCUSSION THREAD

**Discussion by HugoGresse** [directus/directus#15822] [Feature Requests]
I wanted a custom Flow trigger which takes data from a basic `multipart/form-data` and send an email using a third party service. 
However, when using Log to console `{{ $last }}` the body is always empty when using multipart content type. 

```
{
	"path": "/trigger/3dfdxxxxxxxx36",
	"query": {},
	"body": {},
	"method": "POST",
	"headers": {
		"host": "api.fr",
		"connection": "upgrade",
		"content-length": "433",
	"dnt": "1",
	"sec-ch-ua-mobile": "?0",
	"user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36",
	"content-type": "multipart/form-data; boundary=----WebKitFormBoundaryGmxflQJSgBzL9ZEy",
	"accept": "*/*",
	"origin": "http://localhost:5173",
	"sec-fetch-site": "cross-site",
	"sec-fetch-mode": "cors",
	"sec-fetch-dest": "empty",
	"referer": "http://localhost:5173/",
	"accept-encoding": "gzip, deflate, br",
	"accept-language": "en-US,en;q=0.9,fr-FR;q=0.8,fr;q=0.7"
}
```

Any plan to support this or am I missing something?

**Comment by rijkvanzanten** [comment]
The flows endpoint currently only reads the body as `application/json`. There's currently no active plan to support `multipart/form-data` in there yet

**Reply by HugoGresse** [reply]
Supporting multipart would allow to upload file using a regular form and process it using Flow which would allow a lot of possibility!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
